sky_alert/endpoint.py

```diff
@@ -7,6 +14,73 @@
 def healthz():
     return 'OK'
 
-@app.get('/sunrise')
-def sunrise():
-    return '06:00'
\ No newline at end of file
+@app.get('/openweather_sun_data')
+def sun_data(lat: int = 0, lon: int = 0):
+
+    headers = {
+        'Content-Type': 'application/json',
+    }
+
+    params = {
+        "lat": lat,
+        "lon": lon,
+        "appid": os.environ["OPENWEATHER_API_KEY"]
+    }
+
+    api_url = os.environ["OPENWEATHER_API_URL"]
+
+    try:
+        res = requests.get(api_url, params=params, headers=headers)
+        
+        if res.status_code == 200:
+            json_data = res.json()
+            sun_data = get_sunrise_sunset_from_json(json_data)
+            return sun_data
+
+        elif res.status_code == 401:
+            print("error: Unauthorized")
+            return 401
+
+        elif res.status_code == 404:
+            print("error: Resource not found")
+            return 404
+
+        else:
+            print("error: Internal Server Error")
+            return 500
```

Comment:
Maybe out of scope of this PR, but we could consider creating a general error data class to encapsulate these errors + messages, so we don't have to print anything to the console

Reply:
something like 
```
@dataclass
class APIResponse:
    status_code: int 
    message: str
```